.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,20 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v0.2.0
```

Comment:
JFYI we've launched [v1](https://github.com/golangci/golangci-lint-action) with support of caching and without slow docker image pulling, it works much faster